services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -34,17 +43,30 @@ class FlowHelper {
     }
 
     /**
-     * Returns flow endpoint with randomly chosen port and vlan.
+     * Returns flow endpoint with randomly chosen vlan.
+     *
+     * @param useTraffgenPorts whether to try finding a traffgen port
      */
-    private FlowEndpointPayload getFlowEndpoint(Switch sw) {
-        getFlowEndpoint(sw, topology.getAllowedPortsForSwitch(sw))
+    private FlowEndpointPayload getFlowEndpoint(Switch sw, boolean useTraffgenPorts = true) {
+        getFlowEndpoint(sw, topology.getAllowedPortsForSwitch(sw), useTraffgenPorts)
     }
 
     /**
-     * Returns flow endpoint with randomly chosen port and vlan.
+     * Returns flow endpoint with randomly chosen vlan.
+     *
+     * @param allowedPorts from which ports to randomly choose port.
+     * @param useTraffgenPorts whether to try finding a traffgen port. Note that it should be allowed by 'allowedPorts'
```

Comment:
I'd rather rephrase to "Will try to use a port attached to a traffgen if set to true. The port must be present in 'allowedPorts'"